lib/all_your_base_cases.rb

```diff
@@ -57,16 +57,17 @@ def error_message
 
 class AllYourBaseCase::PreProcessor
   class << self
+    attr_reader :row
+
     def call(row)
       @row = row
 
       row.merge('expected' => expected_value)
     end
 
+    private :row
     private
 
```

Comment:
As it is a warning, and not functionally wrong, I would probably leave it at that.  This also means as time goes on and the version of Ruby moves forward, it will eventually take care of itself.